Showcase-iOSTests/GeneratedShowcaseMocks.swift

```diff
@@ -1,4 +1,4 @@
-// MARK: - Mocks generated from file: Showcase-iOS/Contact Us/Interactor/ContactUsInteractable.swift at 2018-05-03 11:41:08 +0000
```

Comment:
Please do not commit this files changes

Reply:
I've updated the .git/info/exclude so that git stops monitoring this file. Thus, from the next PR commits this file won't be tracked anymore.